src/projectscene/view/trackspanel/trackslistmodel.h

```diff
@@ -41,11 +42,11 @@ class TracksListModel : public QAbstractListModel, public muse::async::Asyncable
 
     Q_INVOKABLE void load();
 
-    Q_INVOKABLE void addTrack(TrackTypes::Type type);
-    Q_INVOKABLE void addTracks(TrackTypes::Type type, int quantity);
-    Q_INVOKABLE void duplicateTrack(int index);
-    Q_INVOKABLE void deleteTrack(int index);
-    Q_INVOKABLE void openEffectsForTrack(int index);
+    Q_INVOKABLE void trackAdd(TrackTypes::Type type);
```

Comment:
Previous version was better: verb + noun

Reply:
I can't agree with that.

Reply:
But, i'm okay to stick to either style, but we should enforce it. Right now there is a total mess in [itrackeditinteraction](https://github.com/audacity/audacity/blob/71a065085879ac3f265d66574f796f0729972247/src/trackedit/itrackeditinteraction.h#L34)